go/nbs/dynamo_manifest.go

```diff
@@ -39,13 +40,14 @@ type ddbsvc interface {
 type dynamoManifest struct {
 	table, db string
 	ddbsvc    ddbsvc
+	lock      addr
```

Comment:
Ah. i see the issue you are getting at. The manifests because mutable, where as before they were immutable. Yeah, I think i'd be more comfortable if the lock hash was also owned by the nbs store.